coral/src/app/features/components/filters/SearchFilter.tsx

```diff
@@ -1,23 +1,38 @@
-import { SearchInput } from "@aivenio/aquarium";
+import { Box, Label, SearchInput, Typography } from "@aivenio/aquarium";
 import debounce from "lodash/debounce";
 import type { ChangeEvent } from "react";
 import { useFiltersContext } from "src/app/features/components/filters/useFiltersContext";
 
 type SearchFilterProps = {
+  label: string;
   placeholder: string;
   description: string;
+  ariaDescription: string;
 };
 
-function SearchFilter({ placeholder, description }: SearchFilterProps) {
+function SearchFilter({
+  label,
+  placeholder,
+  description,
+  ariaDescription,
+}: SearchFilterProps) {
   const { search, setFilterValue } = useFiltersContext();
   return (
-    <>
+    <Box maxWidth={"md"}>
```

Comment:
ugh, sorry I did not notice that sooner, but to be safe we should use a id with a unique id in it, like in FileInput

`const inputId = uniqueId("search_field_label_");` 
(uniqueId is from lodash)

In case we have two search filters on a view someday. 

Same for the search-field-description id.

Reply:
[fca1845](https://github.com/Aiven-Open/klaw/pull/1898/commits/fca184516511478245ebccab7926127dd285986f)